src/styles.js

```diff
@@ -51,13 +51,14 @@ export const sizes = {
 
 sizes.phablet = Math.floor((sizes.tablet + sizes.phone) / 2)
 
-export const media = Object.keys(sizes).reduce((accumulator, label) => {
-  accumulator[label] = (...args) => css`
-    @media (max-width: ${sizes[label]}px) {
-      ${css(...args)};
-    }
-  `
-  return accumulator
+export const media = Object.keys(sizes).reduce((acc, s) => {
+  return Object.assign(acc, {
+    [s]: (...args) => css`
+      @media (max-width: ${sizes[s]}px) {
+        ${css(...args)};
+      }
+    `
+  })
 }, {})
```

Comment:
2

Reply:
Yep, all the media width breaks seem the same as in production. BTW one of them seems to narrow to me... Opened #905.